content/en/docs/concepts/architecture/_index.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Kubernetes Architecture"
+title: "Cluster Architecture"
```

Comment:
My only gripe with this is that, interestingly enough, the term "cluster" isn't really defined in the Concepts pages. The "What is Kubernetes" page doesn't mention "cluster" anywhere. The "Kubernetes Components" page mentions cluster plenty of times but doesn't actually define it. The Glossary does have a term for "cluster", though:

> A set of machines, called nodes, that run containerized applications managed by Kubernetes. A cluster has at least one worker node and at least one master node.

So, while I totally agree that "Kubernetes" is redundant, I guess I'd prefer to replace "Cluster" with "System" **or** add a "Concepts -> Cluster" page that explains what we mean by "Cluster" first. Thoughts?

Reply:
Let's add _cluster_ to the glossary, either as well or instead of what you're suggesting @jaypipes

Reply:
Cluster is in the glossary already with the definition above. I guess what I'm pointing out is that the Concepts section doesn't really have anything that says what a cluster is even though the term cluster is used in a number of the Concepts pages. So, perhaps good to modify the "What is Kubernetes" page to define what the term "cluster" refers to, and then it would be good to go forward with this PR change that changes the "Kubernetes Architecture" title to "Cluster Architecture". Hope that makes sense.

Reply:
I can revise this PR to do that.

Reply:
@jaypipes how about this (revised) approach?
If you think it's still short of the mark, please do say so.

Reply:
@sftim It looks like you've addressed the feedback from @jaypipes, and that this PR is headed in the right direction, so I'll `/approve` and @jaypipes can `/lgtm` if he agrees.

/approve